packages/wonder-blocks-dropdown/src/components/single-select.js

```diff
@@ -39,8 +39,53 @@ export type SingleSelectLabels = {|
     someResults: (numOptions: number) => string,
 |};
 
+type DefaultProps = {|
+    /**
+     * Whether this dropdown should be left-aligned or right-aligned with the
+     * opener component. Defaults to left-aligned.
+     */
+    alignment: "left" | "right",
+
+    /**
+     * Whether to auto focus an option.
+     */
+    autoFocus: boolean,
+
+    /**
+     * Whether to enable the type-ahead suggestions feature.
+     *
+     * This feature allows to navigate the listbox using the keyboard.
+     * - Type a character: focus moves to the next item with a name that starts
+     *   with the typed character.
+     * - Type multiple characters in rapid succession: focus moves to the next
+     *   item with a name that starts with the string of characters typed.
+     *
+     * **NOTE:** Type-ahead is recommended for all listboxes, but there might be
+     * some cases where it's not desirable (for example when using a `TextField`
+     * as the opener element).
+     */
+    enableTypeAhead: boolean,
```

Comment:
**suggestion (non-blocking):** Include what the default values for these props are in the docstrings.